Web/stockdogweb/src/containers/Stock/Stock.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import './Stock.css';
+
+import Navbar from '../../components/Navigation/Navbar';
+import Graph from '../../components/Graph/Graph';
+import Article from '../../components/Article/Article';
+import Trade from '../../components/Trade/Trade';
+
+class Stock extends Component {
+   render() {
+      return (
+         <div className="Stock">
+            <Navbar />
+            <Graph />
+            <div className="stock-content">
+               <Article title={"About"} content={'Advanced Micro Devices, Inc. engages in the provision of semiconductor businesses. It operates through the Computing and Graphics and Enterprise, Embedded and Semi-Custom segments.'}/>
```

Comment:
Where are you going to get this content from too?

Reply:
API call

Reply:
State. Lemme change it